spec.md

```diff
@@ -1274,13 +1444,16 @@ Two property types are equivalent if they are the same concrete property type.
 
 # Statements
 
-Statements are used to describe the components within a module and how they
-interact.
+A module body consists of a sequence of statements.
+Statements declare the circuit components and describe their connectivity.
 
 ## Connects
 
-The connect statement is used to specify a physically wired connection between
-two circuit components.
+The components of a module can be connected together using connect statements.
+
+These connections can be both physical, as if the components were connected together with a wire,
+but also logical, allowing metadata such as probes (see [@sec:probe-types;@sec:probe])
+and properties (see [@sec:property-types]) to flow across it during elaboration.
```

Comment:
I'd avoid "physical" and "logical" as that introduces new, undefined concepts. Is there a way to do this by talking about different connections for hardware and non-hardware components?

Reply:
There's no formal distinction, so let's just fudge it until there is.